scripts/cargo/uaa.yml

```diff
@@ -100,31 +100,31 @@ ratelimit:
   credentialID: 'JWT:Claims+"sub"\s*:\s*"(.*?)"'
   limiterMappings:
     - name: AuthToken
-      withCallerRemoteAddressID: 50r/s
+      withCallerRemoteAddressID: 500r/s
```

Comment:
the limits are defined for the voters on concourse
I can understand that you have faster machines, but then maybe you exchange the uaa.yml

Reply:
I have removed this commit